testpilot/base/middleware.py

```diff
@@ -19,7 +19,7 @@ def process_request(self, request):
 
     def _build_extra_meta(self, request):
         td = datetime.datetime.utcnow() - request._logging_start_dt
-        t = int(td.total_seconds() * 10**6)
+        t = int(td.total_seconds() * 1000) # in ms
```

Comment:
I see what you're saying, but that's for the `Timestamp` field.  This is `t` which is defined further down the page in milliseconds